src/glossary.md

```diff
@@ -1,5 +1,29 @@
 # Glossary
 
+## Terms used in the Specification
```

Comment:
I'm back and forth on this.

The main thing I would put in the section would be prescriptive terms, rather than semantic terms (IE. terms used to express the requirements of the spec, not terms that refer to a construct). IE. a term like "function" wouldn't go in this section, but would go into the rest of the glossary.

Reply:
Might be something to discuss tomorrow.